nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/status-history/status-history.component.html

```diff
@@ -170,7 +170,7 @@ <h2 mat-dialog-title>Status History</h2>
             <mat-dialog-actions align="start">
                 <div class="flex w-full">
                     <div class="flex-1">
-                        @if (instances.length > 0) {
+                        @if (instances.length === 0 || fieldDescriptors.length === 0) {
```

Comment:
this is inverted. should be 
```suggestion
                        @if (instances.length > 0 && fieldDescriptors.length > 0) {
```